.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,56 @@
+name: enclave-cc e2e tests
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator enclave-cc e2e tests
+    runs-on: ubuntu-22.04
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
+
+      - name: Build and push to local registry
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: localhost:5000/cc-operator:latest
+
+      - name: Setup kind cluster
+        run: |
+          kind create cluster -n coco-sgx --config tests/e2e/enclave-cc-kind-config.yaml
+          kubectl label node coco-sgx-worker node-role.kubernetes.io/worker=
+
+      - name: Deploy operator from the local registry
+        run: |
+          cd config/default
+          kustomize edit set image localhost:5000/cc-operator:latest
+          kubectl apply -k .
+          sleep 1
+          kubectl wait --for=jsonpath='{.status.conditions[0].status}'=True deployments/cc-operator-controller-manager -n confidential-containers-system
+
+      - name: Install enclave-cc sim
+        run: |
+          kubectl apply -f config/samples/ccruntime-enclave-cc-sim.yaml
+          sleep 1
+          kubectl wait --for=jsonpath='{.status.runtimeClass}'=enclave-cc ccruntime/ccruntime-enclave-cc-sim --timeout=90s
```

Comment:
@fidencio I updated this line and added `--timeout=90s`. The default is 30s and I saw some flakiness with it so I tripled the value (FWIW, the latest run gives 27s).